packages/oasis-actions/src/strategies/ajna/open.ts

```diff
@@ -0,0 +1,78 @@
+import BigNumber from 'bignumber.js'
+import * as ethers from 'ethers'
+
+import ajnaProxyActionsAbi from '../../abi/ajna/ajnaProxyActions.json'
+import { AjnaPosition } from '../../helpers/ajna'
+import { Address, Strategy } from '../../types/common'
+import * as views from '../../views'
+
+export interface OpenArgs {
+  poolAddress: Address
+  dpmProxyAddress: Address
+  quoteAmount: BigNumber
+  quoteTokenPrecision: number
+  collateralAmount: BigNumber
+  collateralTokenPrecision: number
+  price: BigNumber
```

Comment:
Is it the price for what?

Reply:
I need to remove it any way, this one behaves like a slippage 😅